### DISCUSSION TITLE
Further Reading - references

### DISCUSSION THREAD

**Discussion by EduardVasilenko** [becarpenter/book6#47] [General]
Hi Brian,
What if to replace 2 ISOC references to the specific books of one author
with the general references to the ISOC root for all IPv6 books and articles https://www.internetsociety.org/deploy360/ipv6/
Ed/

**Comment by becarpenter** [comment]
My concern is that the ISOC material is not actively maintained, and they did not seem interested in supporting our effort. So I'm not sure that it is a reliable reference.

**Comment by becarpenter** [comment]
I understand there is now work in progress to update the ISOC IPv6 portal. If this happens, we will of course cross-link to it. For now, we can close this discussion.
